### PR TITLE
Destination nav changes

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> New navigation for the destination section.

## Type of documentation

- [ ] New documentation
- [ ] Documentation update
- [ ] Hotfix
- [x] Structural changes

### CODE REVIEW

File: docs/resources/index.mdx [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ---
-#slug: "/docs/community"
 title: "Community"
 description: >-
-  Overview page on how to get help, participate in the community, and whatnot.
```
Comment (pfd) [inline docs/resources/index.mdx:5]: 🤦🏻 sorry

File: src/sidebar.js [hunk 2]
```diff
@@ -654,894 +654,824 @@ export const jsonData = [
...
+      }, {
+        key: "trengo",
+        title: "Trengo",
+        link: "/tmp-destinations/streaming-destinations/business-messaging/trengo/",
+        content: []
+      },
+      {
+        key: "appcenter",
+        title: "Visual Studio App Center",
+        sectionTitle: "Continuos Integration",
```
Comment (pfd) [inline src/sidebar.js:1065]: _Continuous_

**Review by pfd — APPROVED** [review]
Looks awesome. I like how you used the inner section title for the destinations. I found a few broken links for Amazon Redshift, google-bigquery, microsoft sql server but it looks like you are still working folders out with tmp-destinations folder.

File: docs/resources/index.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/resources/index.mdx:5]: yes, @pfd , still working on fixing the file names etc. will organize everything and then push these changes.

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**